railties/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+*   Facilitate use of any regular ERB in database.yml when creating rake tasks.
+
+    Any ERB can be used in `database.yml` regardless of whether it accesses
+    environment configurations.
+
+    *Eike Send*
```

Comment:
We don't need a changelog for Railties in this case, just active record. Can you remove this one?

Reply:
Done!